lib/beaker-pe/install/pe_utils.rb

```diff
@@ -325,6 +325,26 @@ def fetch_pe_on_mac(host, opts)
           end
         end
 
+        # Determine the build package to download on a sles-11 (Intel) host, install that package onto the host.
+        # Assumed file name format: puppet-agent-7.29.1.26.gf344eeefa-1.sles11.x86_64.rpm.
+        # This method should be called after puppet is installed on the master since it relies on the master
+        # telling it the puppet agent version to form the download URL.
+        # @param [Host] host The sles-11 host to download and install the package on.
+        # @param  [Hash{Symbol=>Symbol, String}] opts The options
+        # @api private
+        def install_rpm_on_sles11_host(host, opts)
+          puppet_agent_ver = get_puppet_agent_version(host, opts)
```

Comment:
This is already available in the `opts` hash passed into this method, see https://github.com/puppetlabs/beaker-pe/blob/6f7743054671033a5a22348f1b6d2d4529b02349/lib/beaker-pe/install/pe_utils.rb#L917